bokeh/models/widgets/inputs.py

```diff
@@ -152,6 +152,10 @@ class Slider(InputWidget):
     A callback to run in the browser whenever the current Slider value changes.
     """)
 
+    delay_callback = Bool(default = False, help="""
```

Comment:
No problem.  I can sympathize with the desire to maintain a consistent style. I will try to watch it.